services/ui-src/src/utils/statusing/getRouteStatus.ts

```diff
@@ -13,9 +13,9 @@ export const getRouteStatus = (report: ReportShape): ReportPageProgress[] => {
     formTemplate: { routes },
   } = report;
   // Filter out the reviewSubmit pageType
-  const validRoutes = routes.filter(
-    (r: ReportRoute) => r.pageType !== "reviewSubmit"
-  );
+  const validRoutes = routes
+    .filter((r: ReportRoute) => r.pageType !== "reviewSubmit")
+    .filter((r: ReportRoute) => r.name !== "General Information");
```

Comment:
I think we discussed in refinement that this _should_ still count for the SAR. WP doesn't have questions on the first page, but SAR does. Am I remembering correctly?

Reply:
Ah, I think so. Let me clarify with @jessabean